INSTALL.md

```diff
@@ -207,3 +207,11 @@ https://packages.fedoraproject.org/pkgs/rofi-wayland/rofi-wayland/
 ### NixOS
 
 rofi-wayland is integrated in nixpkgs master ([PR](https://github.com/NixOS/nixpkgs/pull/150169))
+
+## Gentoo
```

Comment:
Just some formatting tweaks:

- please preserve alphabetical order
- it should be `###` and not `##`, to be on the same level and the other distributions
- markdown technically requires more spacing: please add empty lines around lists and after the "Gentoo" sub-title.